prijateli_tree/app/templates/admin_dashboard.html

```diff
@@ -14,6 +14,14 @@
       // Used to clear the URL of parameters after the banner alert is closed.
       window.history.replaceState(null, '', window.location.pathname);
     }
+
+    const copyToClipboard = async (id) => {
```

Comment:
Copies the new URL to the clipboard, so you can paste it elsewhere.

Reply:
Nice.